src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -369,6 +369,7 @@ public string Name
         ///     Collection of PortModels representing all Input ports.
         /// </summary>
         [JsonProperty("Inputs")]
+        [Obsolete("Property will be deprecated in Dynamo 3.0")]
```

Comment:
I don't think we can apply this to the setter specifically, which is why I suggested using our custom `IsObsolete` attribute only on the setter. I don't think we want to obsolete the getter.